### DISCUSSION TITLE
Extention or Desktop app intigration.

### DISCUSSION THREAD

**Discussion by kakalpa** [padloc/padloc#309] [Q&A] [answered]
Hi,

I'm glade I found this tool and it is being working perfectly on my self hosted server. I would like to take it to future by configuring **browser extension** and **desktop application** to go with my server is it possible to do so ? if yes a guide  may be highly appreciated

**Comment by BrunoBernardino** [comment]
Thanks @kakalpa and sorry for the slow reply. This is possible and will be easier to write up a guide for once we release v4 because it'll have automations and simpler commands to make that easier.

**Comment by BrunoBernardino** [comment, marked as answer]
We've published guides for these here: https://github.com/padloc/padloc/tree/main/docs/examples/hosting/app
